src/calculateRentalCost.js

```diff
@@ -5,6 +5,24 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  const NORMAL_PRICE = 40;
+  const MID_TERM = 3;
+  const MID_TERM_DISC = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISC = 50;
+
+  let finalPrice = days * NORMAL_PRICE;
```

Comment:
Consider using `const` instead of `let` for `finalPrice` since its value is not being reassigned, which will enforce immutability.